assets/queries/terraform/aws/user_data_contains_encoded_private_key/query.rego

```diff
@@ -12,5 +14,23 @@ CxPolicy[result] {
 		"issueType": "IncorrectValue",
 		"keyExpectedValue": sprintf("aws_launch_configuration[%s].user_data_base64 doesn't contain RSA Private Key", [name]),
 		"keyActualValue": sprintf("aws_launch_configuration[%s].user_data_base64 contains RSA Private Key", [name]),
+		"searchLine": common_lib.build_search_line(["resource", "aws_launch_configuration", name, "user_data_base64"], []),
+	}
+}
+
+CxPolicy[result] {
+	module := input.document[i].module[name]
+	keyToCheck := common_lib.get_module_equivalent_key("aws", module.source, "aws_launch_configuration", "user_data_base64")
+	user_data := module.user_data_base64
+	not user_data == null
+	contains(user_data, "LS0tLS1CR")
```

Comment:
we should use base64.decode + regex.match to restrict this check

https://www.openpolicyagent.org/docs/latest/policy-reference/#encoding
https://www.openpolicyagent.org/docs/latest/policy-reference/#regex